packages/wrangler/src/bundle.ts

```diff
@@ -77,15 +77,25 @@ export function isBuildFailure(err: unknown): err is esbuild.BuildFailure {
  * Rewrites esbuild BuildFailures for failing to resolve Node built-in modules
  * to suggest enabling Node compat as opposed to `platform: "node"`.
  */
-export function rewriteNodeCompatBuildFailure(err: esbuild.BuildFailure) {
+export function rewriteNodeCompatBuildFailure(
+	err: esbuild.BuildFailure,
+	forPages = false
+) {
 	for (const error of err.errors) {
 		const match = nodeBuiltinResolveErrorText.exec(error.text);
 		if (match !== null) {
+			const issue = `The package "${match[1]}" wasn't found on the file system but is built into node.`;
+
+			const instructionForUser = `${
+				forPages
+					? 'Add the "nodejs_compat" compatibility flag to your Pages project'
+					: 'Add "node_compat = true" to your wrangler.toml file'
```

Comment:
@Skye-31 thanks a lot for the comment, I _think_ you're right, I wasn't really really looking into updating that error message but just adding the Pages version (as you can see the non-pages error message is totally untouched)

I can also see that `node_compat` is used in a [handful of places](https://github.com/search?q=repo%3Acloudflare%2Fworkers-sdk%20%2Fnode_compat%2F&type=code) still so it might just be worth it to change all that in a followup PR instead and keep this one focused only on they Pages related changes?

Also I am not totally sure if here `node_compat` is used for a specific reason 🤷 

Anyways I am happy to change it or keep it and do the cleanup at some other point 🙂

Reply:
I think `node_compat` is just used here because the error message was written before the newer nodejs_compat work. Might be worth updating in a future PR!

Reply:
I just checked the "handful of places" I mentioned above and it seems like the only place that needs fixing is the error message I changed here 😓 🤦 (sorry should have checked the search result more carefully the first time around)

I'll open a PR to fix that 🙂